builders/interoperability/mrl.md

```diff
@@ -0,0 +1,152 @@
+
+xcGLMR must be used as a fee token in `xTokens.transferMultiassets` as opposed to the ERC-20 token because of a restriction with how the xTokens pallet currently works.  
+
+??? code "xTokens.transferMultiassets Extrinsic Construction" 
+    ```js
+    --8<-- 'code/mrl/transfer-multiassets.js'
+    ```
+
+To generate the second end of the transaction (remote EVM call), you need both the EVM transaction and the XCM instructions that executes said EVM transaction. The EVM transaction can be constructed as a transaction that interacts with the [batch precompile](/builders/pallets-precompiles/precompiles/batch){target=_blank} so that two transactions can happen in one. This is helpful because this EVM transaction has to both approve the xLabs relayer to relay the ERC-20 token as well as relay action itself.  
+
+??? code "Batched EVM Construction" 
+    ```js
+    --8<-- 'code/mrl/evm-tx.js'
+    ```
+
+You would want to send a message such that the `Transact` XCM instruction gets successfully executed. The most common method to do this is through `polkadotXcm.send` and sending the `WithdrawAsset`, `BuyExecution`, and `Transact` instructions. `RefundSurplus` and `DepositAsset` can also be used to ensure no assets get trapped, but they are technically optional.
+
+??? code "polkadotXcm.send Extrinsic Construction" 
+    ```js
+    --8<-- 'code/mrl/polkadotxcm-send.js'
+    ```
+
+To ensure that both the `xTokens.transferMultiassets` and the `polkadotXcm.send` transactions are sent together, you can batch them together using `utility.batchAll`. At time of writing, this helps ensure that the assets transfer happens before the EVM transaction, a necessary distinction. Unfortunately, this is subject to change with future XCM updates.  
+
+??? code "utility.batchAll Extrinsic Construction and Execution"
+    ```js
+    const batchExtrinsic = originChainPolkadotJsAPI.tx.utility.batchAll([
+        xTokensExtrinsic,
+        xcmExtrinsic
+    ]);
+
+    // Send batch transaction
+    return await batchExtrinsic.signAndSend(account, ({ status }) => {
+        if (status.isInBlock) console.log(`Moonbase Beta transaction finished!`);
+    });
+    ```
+
+If you would like to see an example script that fully implements this, an example is available in a [GitHub repository](https://github.com/jboetticher/mrl-reverse){target=_blank}.  
+
+It’s important to note that not every parachain will have xTokens and the other pallets implemented in a way that will allow this path. Substrate-based chains are very flexible, to the point where a standard doesn’t exist. If you believe your parachain does not support this path, please provide an alternative solution in the Moonbeam forum and to the Wormhole team.  
```

Comment:
is there a preferred contact method for the Wormhole team?

Reply:
Not sure. You could reach out directly through telegram or discord, but the best way is potentially through us instead of directly to the Wormhole team.